### PR TITLE
Move pull linux-docs job to Ubuntu 20.04

### PR DESCRIPTION
**Pull request by kit1980**

### CODE REVIEW

File: .circleci/docker/build.sh [hunk 1]
```diff
@@ -250,6 +250,15 @@ case "$image" in
     VISION=yes
     ROCM_VERSION=5.1.1
     ;;
+  pytorch-linux-focal-py3.7-gcc7)
```
Comment (suo) [inline .circleci/docker/build.sh:253]: fyi you'll need to add this to the docker build workflow as well

Comment (kit1980) [reply .circleci/docker/build.sh:253]: This particular image is already in the docker build workflow.

**Review by janeyx99 — APPROVED** [review]

**Pull request closed without merging**